### DISCUSSION TITLE
Nuget Deprecation

### DISCUSSION THREAD

**Discussion by pietrocarpinacci-dufercodev** [bagetter/BaGetter#137] [Q&A] [answered]
Hi,
is there any way to deprecate a specific version of a nuget?
Visual studio now show an alert if you are using a deprecated nuget library from nuget.org. It could be replicated on bagetter?

Thanks,
Pietro

**Comment by Regenhardt** [comment, marked as answer]
Bagetter currently doesn't support deprecation, but I'd definitely like to add it in the future. It's been a feature request in BaGet for a long time too.

**Reply by pietrocarpinacci-dufercodev** [reply]
It could be great to have this feature!
